mitodl/ocw-studio
# update import code and test data to reflect latest ocw-to-hugo changes

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/521

#### What's this PR do?
In https://github.com/mitodl/ocw-to-hugo/pull/331 and https://github.com/mitodl/ocw-to-hugo/pull/332 we updated the output structure of `ocw-to-hugo` to better match the way `ocw-studio` exports Hugo content for building course sites.  

#### How should this be manually tested?
There isn't a great way to test this until https://github.com/mitodl/ocw-hugo-themes/pull/162 is released, at which point the import code in the currently deployed versions of `ocw-studio` will not work anymore.  The release of the above PR will trigger a full rebuild of all imported course site markdown and upload to the `ocw-to-hugo` S3 buckets.  Once this is complete, this PR can be tested using the following steps:

 - If you've never imported course sites using `import_ocw_course_sites` before, read the readme section on it and make sure you're configured to download course data from S3
 - Spin up your local instance of `ocw-studio` and import a course with `docker-compose run web ./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --filter 18-06sc`
 - Visit your local instance of `ocw-studio` and verify that the course was successfully imported
